website/docs/r/nas_access_rule.html.markdown

```diff
@@ -13,7 +13,9 @@ Provides a Nas Access Rule resource.
 
 When NAS is activated, the Default VPC Permission Group is automatically generated. It allows all IP addresses in a VPC to access the mount point with full permissions. Full permissions include Read/Write permission with no restriction on root users.
 
--> **NOTE:** Available in v1.34.0+.
+-> **NOTE:**
+- Available in v1.34.0+.
+- Deprecated in v1.189.0+, alicloud_all_nas_access_rule is recommended in v1.190.0+.
```

Comment:
Deprecated in v1.190.0, alicloud_all_nas_access_rule is recommended in v1.190.0+.